src/main/java/io/stargate/sgv2/jsonapi/config/constants/DocumentConstants.java

```diff
@@ -8,31 +8,40 @@ interface Fields {
     /** Primary key for Documents stored; has special handling for many operations. */
     String DOC_ID = "_id";
 
-    /**
-     * Atomic values are added to the array_contains field to support $eq on both atomic value and
-     * array element
-     */
-    String DATA_CONTAINS = "array_contains";
```

Comment:
Reordering to keep fields (of Collection Documents, JSON) and columns (actual backing DB columns) grouped, ordered. Not removed.